conda/cli/conda_argparse.py

```diff
@@ -115,7 +120,9 @@ def do_call(arguments: argparse.Namespace, parser: ArgumentParser):
     Serves as the primary entry point for commands referred to in this file and for
     all registered plugin subcommands.
     """
-    # First, check if this is a plugin subcommand; if this attribute is present then it is
+    if context._no_plugins:
```

Comment:
Here we want to reference the special property we created:

```suggestion
    if context.no_plugins:
```